### DISCUSSION TITLE
Issue when mapping i to j in Visual mode

### DISCUSSION THREAD

**Discussion by aviv-akhansari** [LazyVim/LazyVim#2928] [Q&A] [answered]
Hi,

I have a keymap like this:
```lua
vim.keymap.set({ "n", "v" }, "i", "j", { desc = "Down" })
```

But during the Visual mode, when I quickly press `i` more than twice, either nothing happens or it selects an entire bloc (I think it's `<leader>ii`).
I have to press `i`, wait a bit, and then press `i` again to be able to move the cursor down multiple times.

How can I fix this?

**Comment by dpetka2001** [comment]
`i` is used by several plugins for accepting text-objects motions when in Visual mode. To achieve what you want, you could do
```lua
  { "echasnovski/mini.ai", enabled = false },
  { "echasnovski/mini.indentscope", enabled = false },
  { "which-key.nvim", opts = { plugins = { presets = { text_objects = false } } } },
```
Note that this will disable some functionality of LazyVim default settings and it's up to you to decide whether you can do without that or you would prefer to map `j` to some other key that isn't used for such operations (like `i` is).

**Reply by aviv-akhansari** [reply, marked as answer]
Oh you saved me from pain, thanks.
I could fix it like this:

```lua
vim.keymap.set({ "n", "v" }, "i", "gj", { desc = "Down" })
```

```lua
  {
    "echasnovski/mini.indentscope",
    opts = {
      mappings = {
        object_scope = "li",
      },
    },
  },
```
